src/QMCWaveFunctions/Jastrow/CountingJastrowBuilder.cpp

```diff
@@ -0,0 +1,190 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2016 Jeongnim Kim and QMCPACK developers.
+//
+// File developed by: Luke Shulenburger, lshulen@sandia.gov, Sandia National Laboratories
+//
+// File created by: Luke Shulenburger, lshulen@sandia.gov, Sandia National Laboratories
+//////////////////////////////////////////////////////////////////////////////////////
+
+#include "QMCWaveFunctions/Jastrow/CountingJastrowBuilder.h"
+#include "QMCWaveFunctions/Jastrow/CountingJastrowOrbital.h"
+#include "QMCWaveFunctions/Jastrow/DiffCountingJastrowOrbital.h"
+#include "Utilities/ProgressReportEngine.h"
+#include <iostream>
+
+namespace qmcplusplus
+{
+
+CountingJastrowBuilder::CountingJastrowBuilder(ParticleSet& target, TrialWaveFunction& psi):
+  WaveFunctionComponentBuilder(target, psi)
+  {
+  ClassName="CountingJastrowBuilder";
+  NameOpt="0";
+  TypeOpt="unknown";
+  RegionOpt="normalized_gaussian";
+  SourceOpt=targetPtcl.getName();
+}
+
+template<class precision, template<class> class CountingRegionType>
+class CountingJastrowTypeHelper
```

Comment:
Can we remove this class and directly put its content at its use? It seems it is only used once.